.github/workflows/publish-release-type.yaml

```
@@ -57,7 +57,7 @@ jobs:
           channel: team-pitch-black
```

these messages should print the release versions too as mentioned in the screenshot below ✨

---

adjusted

---

i cant seem to comment on the above line 
```suggestion
          message: ":tada: ${{ steps.get_version.outputs.VERSION }} released!"
```
i meant we need those versions in the team pitch-black channel as well so we know our releases were ok or bad

---

Added